src/utils/StripedLoader.js

```diff
@@ -4,7 +4,9 @@ import { loading } from '../styles/generic'
 
 let StripedLoader = (Component, style = {}) =>
   observer(props => (
-    <div style={{ ...style, ...(props.loading && loading) }}>
+    <div
+      style={{ ...style, ...(props.node && props.node.updating && loading) }}
```

Comment:
`StripedLoader` was written to be a generic loader and shouldn't directly depend on `updating` as that ties it to contexture